services/src/api/handlers/wcs.rs

```diff
@@ -493,23 +493,42 @@ fn default_time_from_config() -> TimeInterval {
 
 #[cfg(test)]
 mod tests {
-    use crate::contexts::{Session, SessionContext, SimpleApplicationContext};
-    use crate::util::tests::with_temp_context;
-    use crate::util::tests::with_temp_context_from_spec;
-    use crate::util::tests::{read_body_string, register_ndvi_workflow_helper, send_test_request};
+    use crate::contexts::{ApplicationContext, Session, SessionContext};
+    use crate::pro::contexts::ProPostgresContext;
+    use crate::pro::ge_context;
+    use crate::pro::users::UserAuth;
+    use crate::pro::util::tests::add_ndvi_to_datasets;
+    use crate::util::tests::{read_body_string, send_test_request};
+    use crate::workflows::registry::WorkflowRegistry;
+    use crate::workflows::workflow::Workflow;
     use actix_web::http::header;
     use actix_web::test;
     use actix_web_httpauth::headers::authorization::Bearer;
     use geoengine_datatypes::raster::{GridShape2D, TilingSpecification};
-    use geoengine_datatypes::util::test::TestDefault;
-
-    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
-    async fn get_capabilities() {
-        with_temp_context(|app_ctx, _| async move {
-        let ctx = app_ctx.default_session_context().await.unwrap();
-        let session_id = ctx.session().id();
-
-        let (_, workflow_id) = register_ndvi_workflow_helper(&app_ctx).await;
+    use geoengine_operators::engine::{RasterOperator, TypedOperator};
+    use geoengine_operators::source::{GdalSource, GdalSourceParameters};
+    use tokio_postgres::NoTls;
+
+    #[ge_context::test]
+    async fn get_capabilities(app_ctx: ProPostgresContext<NoTls>) {
+        let session = app_ctx.create_anonymous_session().await.unwrap();
+        let ctx = app_ctx.session_context(session.clone());
+
+        let session_id = session.id();
+
+        let (_gdal_dataset_id, gdal_dataset_name) =
+            add_ndvi_to_datasets(&app_ctx, true, true).await;
```

Comment:
why not use the register_ndvi_workflow_helper anymore?

Reply:
I think there are several more occurrences you can replace. when I search for "add_ndvi_to_datasets(&app_ctx, true, true).await;" there are multiple hits in wms.rs

Reply:
[7599190](https://github.com/geo-engine/geoengine/pull/1006/commits/7599190757699f534cc0e138d64ad9c54e5b5e25)